recipe/migrations/root_base63406.yaml

```
@@ -0,0 +1,8 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for root_base 6.34.06
+  kind: version
+  migration_number: 1
+migrator_ts: 1753378571.7182064
+root_base:
+- 6.34.06
```

Waiting for https://github.com/conda-forge/root-feedstock/pull/301 & https://github.com/conda-forge/root-feedstock/pull/300 to turn this into

```suggestion
- 6.34.10
- 6.36.02
```

---

That is possible now. Just FYI, I already started the PRs for 6.36.04 at https://github.com/conda-forge/llvmdev-feedstock/pull/342 and https://github.com/conda-forge/clangdev-feedstock/pull/375

---

Well, see if you can upgrade root to 6.36.04, and then we can do a migration for the most up-to-date builds. :)

---

Yes, I'm hopeful the changes won't be major and the upgrade will be faster, I'll keep you updated 👍

---

6.36.04 is available on conda-forge, I believe you can go ahead with this modification :+1:

---

Another question though. IIUC, if this gets merged with the migration to `6.34.10`, it will coincidentally also fulfill the second alternative described by https://github.com/conda-forge/root-feedstock/pull/307#issuecomment-3245991117, right? i.e. the downstream package will get the latest build of ROOT 6.34.10 which also has the fix mentioned in that issue. Otherwise, I am happy to create a new migration. I just want to make sure I'm understanding how this works since I've never done it before

---

Yes, the migration would build for both 6.34.10 as well as 6.36.04, so that should be fine.